packages/cashier/src/Components/recent-transaction.jsx

```diff
@@ -0,0 +1,114 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Button, Text, Icon } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { epochToMoment } from '@deriv/shared';
+import { connect } from 'Stores/connect';
+
+const RecentTransaction = ({ crypto_transactions, currency }) => {
+    const getStatus = status => {
+        switch (status) {
+            case 'CONFIRMED':
+                return 'Successful';
+            case 'LOCKED':
+                return 'In review';
+            case 'PERFORMING_BLOCKCHAIN_TXN':
+                return 'In process';
+            case 'PROCESSING':
+                return 'In process';
+            case 'SENT':
+                return 'Successful';
+            case 'VERIFIED':
+                return 'In process';
```

Comment:
Do you think we can move these to constants?

Reply:
Also do the returns need to be `localized`?

Reply:
Yep,  I will localize it, not sure how exactly you want it to move to constants

Reply:
Create a file like `transaction-status.js` in `packages/cashier/src/Constants`.